src/style.css

```
@@ -1 +1,26 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.borders {
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star--active {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
```

background-repeat: no-repeat already settled.
use stars--1 ... stars--5 modifiers for show active stars
```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
...
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(images/star-active.svg);
```